src/courtroom.cpp

```diff
@@ -2971,7 +2976,7 @@ void Courtroom::log_ic_text(QString p_name, QString p_showname,
   chatlogpiece log_entry(p_name, p_showname, p_message, p_action, p_color);
   ic_chatlog_history.append(log_entry);
   if (ao_app->get_auto_logging_enabled() && !ao_app->log_filename.isEmpty())
-    ao_app->append_to_file(log_entry.get_full(), ao_app->log_filename, true);
+    ao_app->append_to_file("[IC]" + log_entry.get_full(), ao_app->log_filename, true);
```

Comment:
Meh. Marking OOC explicitly is fine, but I think it's too verbose to mark all IC chat explicitly as such.

```suggestion
    ao_app->append_to_file(log_entry.get_full(), ao_app->log_filename, true);
```

Reply:
Easier to filter that way